tests/Feature/controller/AddValidationTest.php

```diff
@@ -11,138 +11,200 @@ class AddValidationTest extends TestCase
 {
     use DatabaseMigrations;
     use WithoutMiddleware;
+
```

Comment:
Thanks this test case looks good. It only tests the negative cases though. So please add a positive test case as well (where everything is validated OK and does get in the database)